src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -2276,7 +2282,7 @@ namespace HVACUnitaryBypassVAV {
                                            DataHVACGlobals::ContFanCycCoil,
                                            PartLoadFrac,
                                            OnOffAirFlowRatio);
-                        if (DataLoopNode::Node(CBVAV(CBVAVNum).DXCoilInletNode).Temp < CBVAV(CBVAVNum).CoilTempSetPoint) {
+                        if (DataLoopNode::Node(CBVAV(CBVAVNum).DXCoilInletNode).Temp <= CBVAV(CBVAVNum).CoilTempSetPoint) {
```

Comment:
similar clean up to avoid turning on the coil at very low PLR if inlet T = CoilTempSP